Tests/XcodeProjTests/Scheme/XCSchemeTests.swift

```diff
@@ -58,6 +58,12 @@ final class XCSchemeIntegrationTests: XCTestCase {
         }
     }
 
+    func test_read_iosScheme_noParallelizable_setsswiftTestingOnly() throws {
+        let scheme = try XCScheme(path: iosSchemePath)
+
+        XCTAssertEqual(scheme.testAction?.testables.first?.parallelizable, .swiftTestingOnly)
+    }
```

Comment:
I haven't realized this is tested [here](https://github.com/tuist/XcodeProj/blob/main/Tests/XcodeProjTests/Scheme/XCSchemeTests.swift#L368) – I think we can remove this test case.